changelog.d/pr-10982

```diff
@@ -0,0 +1,14 @@
+synopsis: Allow reinstalling packages like base and template-haskell for GHC>9.14
```

Comment:
Could you please update the PR title (and description?) according to this? Currently it's very different.